README.md

```diff
@@ -3,25 +3,49 @@ Azure Functions prepared for the Mapping Viz Tools project. This repo contains [
 
 More info on the Mapping Viz Tools [wiki](https://github.com/zooniverse/mapping-viz-tools/wiki/What-are-Azure-Functions%3F)
 
+## Getting started
 ### Dependencies
 In order to run this code, you'll need to install the Azure Functions [core tools.](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=macos%2Ccsharp%2Cbash#v2)
 
 Although not required, you may also want to install the Azure VS Code [extension.](https://code.visualstudio.com/docs/azure/extensions)
 
-### yarn install
+### Local database setup
+1. Install PostgreSQL 9.5 and run the PostgreSQL service locally 
```

Comment:
Not blocking, this can be a real hassle, especially if not on linux / mac OS. In other zooniverse gh repositories we provide a docker-compose.yml to start a postgres server. 

It might be worth adding this here to standardize how we launch dependent services for our code bases, https://github.com/zooniverse/tove/blob/5d6047de4ec8cabb62b6304748e2ea693573288d/docker-compose.yml#L1-L7

Reply:
I can make an issue for this so that if there's further dev work that gets done on this project we can dockerize the database - at the moment, I don't think it's worth the time/effort to dockerize for the sake of the postgres installation, as far as I'm aware none of the devs on our team use Windows; and installing postgres on Mac using homebrew is incredibly simple, probably also very simple on Linux.

Admittedly it would be a good excercise for me to learn to set up docker for an app (or for an app database) but right now it doesn't make sense for me to prioritize this.

Reply:
Let's talk in our Tuesday backend meeting if you disagree/feel that this should be prioritized.

Reply:
i'll defer to you

While i agree brew is pretty easy, sometimes you need other dependencies that may not be installed or taps get deprecated over time. It can end up being a hassle. FWIW i've done this a lot over my time here, having a reproducible build via  docker image can save a lot of time and we've been moving towards using docker for all our repositories